docs/user/dashboard/dashboard.asciidoc

```diff
@@ -228,6 +228,20 @@ To view the data field summary information, navigate to the field, then click *i
 [role="screenshot"]
 image::images/lens_data_info.png[Data summary window]
 
+Depending on the type of data, *Lens* shows a summary that better suits the field analysis: for date types the time distribution, for string types the top 10 values and for numeric types are shown a detailed summary with the top 10 values and a value distribution.
+
+[role="screenshot"]
+image::images/lens_data_info_documents.png[Data summary analyzed documents]
+
+*Lens* uses a sample of 5000 documents to perform the fields analysis, therefore the bottom line of the summary hints how many documents have been analyzed to build the distribution.
+
+When *Lens* presents the top 10 values distribution every other value, together with documents with no data for the inspected field, are shown with the "Other" entry in the distribution chart.
```

Comment:
I don't think "no data" is counted in value_count, did you see a case where we would include it?

```suggestion
When *Lens* presents the top 10 values distribution, it also shows the percentage of "Other" values. For array value fields, the percentage distribution is considers each value in the array separate.
```